packages/common/src/store/pages/saved-page/selectors.ts

```diff
@@ -2,13 +2,30 @@ import { CommonState } from 'store/commonStore'
 
 import { ID } from '../../../models/Identifiers'
 
-import { LibraryCategory } from './types'
+import { LibraryCategory, SavedPageTabs } from './types'
 
 export const getSaved = (state: CommonState) => state.pages.savedPage
 export const getTrackSaves = (state: CommonState) =>
   state.pages.savedPage.trackSaves
-export const getSelectedCategory = (state: CommonState) =>
-  state.pages.savedPage.selectedCategory
+
+export const getCollectionsCategory = (state: CommonState) => {
+  return state.pages.savedPage.collectionsCategory
+}
+
+export const getTracksCategory = (state: CommonState) => {
+  return state.pages.savedPage.tracksCategory
+}
+
+export const getCategory = (
+  state: CommonState,
+  props: { currentTab: SavedPageTabs }
```

Comment:
can this just be a value instead of object? so its like `getCategory(state, tab)`?

Reply:
i guess named object is nice, but just a thought.

Reply:
yeah i was just following the pattern of all the other ones in this file!